### PR TITLE
feat: minimal testnets

### PR DESCRIPTION
**Pull request by estebanmino**
Fixes BX-1059
Figma link (if any):

## What changed (plus any additional context for devs)

added testnets in ticket
user won't be able to switch networks for a session from the extension
apps are gonna be able to switch networks to a testnet
approve txs working
approve signing appear with correct network name
testnet txs are gonna be pending until the tx is confirmed, then the tx will disappear
while a tx is pending you can right click and see the tx in explorer

## Screen recordings / screenshots

https://www.loom.com/share/22e6c711cacb471cb0af7e72adf3f22c

### CODE REVIEW

File: src/core/resources/assets/userTestnetNativeAsset.ts [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+import { ChainId, ChainName, ChainNameDisplay } from '~/core/types/chains';
+import { fetchAssetBalanceViaProvider } from '~/core/utils/assets';
+import { getChain } from '~/core/utils/chains';
+
+const USER_ASSETS_REFETCH_INTERVAL = 60000;
+
+const getNativeAssetMock = ({ chainId }: { chainId: ChainId }) => {
+  const chain = getChain({ chainId });
+  const nativeAssetMock = {
+    address: 'eth',
```
Comment (greg-schrammel) [inline src/core/resources/assets/userTestnetNativeAsset.ts:22]: I think you can use `NATIVE_ASSETS_PER_CHAIN` from `'~/core/references'` to help here

**Review by greg-schrammel — APPROVED** [review]
🔥

**Review by brunobar79 — APPROVED** [review]
🔥 🔥 🔥

File: src/core/resources/assets/userTestnetNativeAsset.ts (hunk 1 above)
Comment (estebanmino) [reply src/core/resources/assets/userTestnetNativeAsset.ts:22]: using and updated NATIVE_ASSETS_PER_CHAIN with testnets

**Pull request merged**